cmd/validatorcmd/getBalance.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package validatorcmd
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanche-cli/pkg/cobrautils"
+	"github.com/ava-labs/avalanche-cli/pkg/networkoptions"
+	"github.com/ava-labs/avalanche-cli/pkg/txutils"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/units"
+	"github.com/spf13/cobra"
+)
+
+var globalNetworkFlags networkoptions.NetworkFlags
+
+var (
+	l1              string
+	subnetID        string
+	validationIDStr string
+)
+
+var getBalanceSupportedNetworkOptions = []networkoptions.NetworkOption{
+	networkoptions.Local,
+	networkoptions.Devnet,
+	networkoptions.EtnaDevnet,
+	networkoptions.Fuji,
+	networkoptions.Mainnet,
+}
+
+func NewGetBalanceCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "getBalance",
+		Short: "Gets current balance of validator on P-Chain",
+		Long: `This command gets the remaining validator P-Chain balance that is available to pay
+P-Chain continuous fee`,
+		RunE: getBalance,
+		Args: cobrautils.ExactArgs(0),
+	}
+
+	networkoptions.AddNetworkFlagsToCmd(cmd, &globalNetworkFlags, true, getBalanceSupportedNetworkOptions)
+	cmd.Flags().StringVar(&l1, "l1", "", "name of L1 (to get balance of bootstrap validators only)")
```

Comment:
this one is confusing to me.

It seems to me that we should just provide a separate command to list bootstrap validators by l1/blockchain name based on the sidecar info. And display their nodeID/validation ID pairs.

this func would just accept nodeiD or validationID to display a balance. I feel like it would give us clean and predictable UX and less prompts and checks. Like we would not fail if `l1` is not provided.

Reply:
if no validatorID/NodeID provided - fail fast